package.json

```
@@ -20,7 +20,7 @@
   "homepage": "https://offchainlabs.com",
   "scripts": {
     "audit:ci": "audit-ci -l -a 1067409 1067487 1067486",
-    "postinstall": "node ./scripts/genAbi.js",
+    "prepare": "$npm_execpath gen:abi",
```

this change breaks with windows computers that aren't using a linux shell
```suggestion
    "prepare": "node ./scripts/genAbi.js",
```